src/olympia/abuse/tests/test_models.py

```diff
@@ -2886,23 +2886,6 @@ def test_report_to_cinder_approve_override_previous_decision_has_cinder_id(self)
             expect_create_override_call=True,
         )
 
-    def _check_notify_emails(self, decision, log_entry):
-        assert len(mail.outbox) == 1
-        assert mail.outbox[0].to == [decision.addon.authors.first().email]
-        assert str(log_entry.id) in mail.outbox[0].extra_headers['Message-ID']
-        assert 'days' not in mail.outbox[0].body
-        assert 'some review text' in mail.outbox[0].body
-        assert 'some policy text' not in mail.outbox[0].body
-        AttachmentLog.objects.create(
-            activity_log=log_entry,
-            file=ContentFile('Pseudo File', name='attachment.txt'),
-        )
-        decision.send_notifications()
-        assert 'An attachment was provided.' not in mail.outbox[0].body
-        assert 'To respond or view the file,' not in mail.outbox[0].body
-        assert 'An attachment was provided.' in mail.outbox[1].body
-        assert 'To respond or view the file,' in mail.outbox[1].body
-
```

Comment:
btw, this is just refactoring because I realised `_check_notify_emails` was only used in one test, so it was confusing - I initially thought I could change here to test multiple tests.  code has been cut&pasted into `test_execute_action_with_action_date_already` directly.